apis/v1alpha1/common.go

```diff
@@ -525,6 +525,18 @@ type LoggingSpec struct {
 	// +optional
 	// +kubebuilder:validation:Enum:={"json", "text"}
 	Format LogFormat `json:"format,omitempty"`
+
```

Comment:
Can we create a dedicated struct to describe the tracing? It should not be part of `Logging{}`.

Reply:
For test each component:
![image](https://github.com/user-attachments/assets/f11fea04-845e-4c07-8f73-bc0bcf47e4f4)